rosdep/base.yaml

```diff
@@ -4626,6 +4626,321 @@ libpcl-all-dev:
   rhel: [pcl-devel]
   slackware: [pcl]
   ubuntu: [libpcl-dev]
+libpcl-apps:
+  debian:
```

Comment:
```suggestion
  arch: [pcl]
  debian:
```

Reply:
please apply to all the other libs this change

Reply:
In my pull request, a check complained that the pcl package could not be found on arch and opensuse, see here: https://github.com/ros/rosdistro/pull/35137#issuecomment-1309476280

Reply:
Thanks for the info.
I was looking at the preexisting `libpcl-all-dev` rule.
We should fix that one later, but no need to do that here.